mevolib/inference/__init__.py

```diff
@@ -144,7 +161,7 @@ def get_phylogeny ( binary, infile, infile_format, args = 'default',
     * The output file format must be supported by Bio.Phylo.
     """
     # Get the variables associated with the given phylogenetic inference tool
-    bin_path, bin_name = os.path.split(binary)
+    bin_name = os.path.split(binary)
```

Comment:
Where possible, change the usage of `os.path` by `pathlib` library [[link](https://docs.python.org/3/library/pathlib.html)].